### DISCUSSION TITLE
Does anyone use this rtos in production stage ??

### DISCUSSION THREAD

**Discussion by ahmedalkabir** [kmilo17pet/QuarkTS#24] [Q&A] [answered]
Hey, I'm just porting this rtos to atmega328p MCU, and so far the process is going well.
My Question is Did anyone use it in products ??
btw, it's a good project and I'll contribute to it as soon as finish my tasks😁  .
@kmilo17pet @julianbustamante7 @codacy-badger

**Comment by kmilo17pet** [comment, marked as answer]
hi @ahmedalkabir, the OS is developed using a formal and rigorous process targered to safe critical applications.  We normally used in small cores or when a full preemptive-RTOS its not required., the ATMEGA328p its a good target for this RTOS. As far i know by colleagues, right now is being used in production and also its currently running in a wide range of products without any problerm reported here. So, you can confidently use it in production.

**Reply by ahmedalkabir** [reply]
Oh, I see, well I started to use this so far it's quite good, in the next step I will use it with cortex m0+.
and thanks for responding

**Reply by fbucafusco** [reply]
Hi @kmilo17pet :
I am from Argentina (I kept this post in english, though).
Do you have any repo with examples for this OS? That could be interesting for learning purposes. 
Thanks!

**Reply by ahmedalkabir** [reply]
hey @fbucafusco, after working on this OS, I thought about providing examples for learning purposes, I'll create a repository for this as soon as possible.

**Reply by fbucafusco** [reply]
Great!! 
Thanks!

**Comment by thejbte** [comment]
Hi, @fbucafusco <https://github.com/fbucafusco>, @ahmedalkabir
<https://github.com/ahmedalkabir> .
I share this example with you, it is only a small part of a project that I
did a while ago, it has all the configuration of the peripherals with the
LL, usb (HAL), it also has a temperature sensor (i2c), AT commands and uses
the QuarkTS

https://github.com/thejbte/Ejemplo_QuartKTS__STM32

best regards.
*Julián Bustamante Narvaez. - *My*LinkedIn*
<https://www.linkedin.com/in/julian-bustamante-narvaez-3a729912a/>
-*Ingeniero en Instrumentación y control.*
*-Especialista en sistemas embebidos.*
*-Docente cátedra - **Politécnico Colombiano Jaime Isaza Cadavid*
<http://www.politecnicojic.edu.co/>
*-Embedded applications engineer - Tecrea S.A.S. <https://tecrea.com.co/>*
*-Certified LabVIEW Associate Developer (CLAD)*

**Reply by fbucafusco** [reply]
thanks!
